test/python/circuit/test_circuit_operations.py

```diff
@@ -1550,6 +1551,82 @@ def instructions():
         self.assertEqual(circuit, expected)
         self.assertEqual(circuit.name, "test")
 
+    def test_circuit_has_control_flow_op(self):
```

Comment:
I would expand this to cover a circuit with each control flow op type and make sure it's detected. The set of defined operations is here: https://github.com/Qiskit/qiskit/blob/main/qiskit/circuit/controlflow/__init__.py#L28

Reply:
Addressed in 98c2e2c